lib/fencing/st_client.c

```diff
@@ -2513,3 +2513,29 @@ stonith__parse_targets(const char *target_spec)
     }
     return targets;
 }
+
+/*!
+ * \internal
+ * \brief Determine if a later stonith event succeeded.
+ */
+gboolean
+stonith__later_succeeded(stonith_history_t *event, stonith_history_t *top_history)
```

Comment:
This assumes the history has been sorted per sort_stonith_history(), so that needs to be documented and any other callers have to follow suit